hail/python/hail/vds/variant_dataset.py

```diff
@@ -18,6 +18,7 @@ def read_vds(
     _assert_reference_type=None,
     _assert_variant_type=None,
     _warn_no_ref_block_max_length=True,
+    _drop_end=False,
```

Comment:
Could add this to the parameters section of the docs string below

Reply:
Private parameter, existing _only_ to make life easier in the Variant Dataset Combiner.

For education, the VDS combiner is our data production solution for taking GVCFs (single sample VCF with extra data for reference blocks, produced by GATK HaplotypeCaller) and/or smaller VariantDatasets and merging them into one large VDS.